MODULE.bazel

```diff
@@ -0,0 +1,51 @@
+module(
+    name = "io_bazel_rules_docker",
+    version = "0.26.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "platforms", version = "0.0.4")
+bazel_dep(name = "bazel_skylib", version = "1.3.0")
+bazel_dep(name = "rules_go", version = "0.35.0", repo_name = "io_bazel_rules_go")
+bazel_dep(name = "gazelle", version = "0.28.0", repo_name = "bazel_gazelle")
+
+go_deps = use_extension("@bazel_gazelle//:extensions.bzl", "go_deps")
+go_deps.from_file(go_mod = "//:go.mod")
+use_repo(
+    go_deps,
+    "com_github_ghodss_yaml",
+    "com_github_google_go_containerregistry",
+    "com_github_kylelemons_godebug",
+    "com_github_pkg_errors",
+    "in_gopkg_yaml_v2",
+)
+
+register_toolchains(
+    # Register the default docker toolchain that expects the 'docker'
+    # executable to be in the PATH
+    "//toolchains/docker:default_linux_toolchain",
+    "//toolchains/docker:default_windows_toolchain",
+    "//toolchains/docker:default_osx_toolchain",
+)
+
+# FIXME @aignas 2023-01-03: I am not sure yet how the user could customize this.
```

Comment:
Will probably need fixing?

Reply:
What do they need to customise?

Reply:
There is a note in the `README.md` about the users being able to configure the docker toolchain, however, I am not sure if the way the dependency chain between the default toolchains and the docker config was the best way to do that. Because the user has no way to configure the default toolchains that are registered by the module, it would be best if the toolchains for each platform could be defined in the repository created by the configuration rule.

I tried to port the comment from README in https://github.com/bazelbuild/rules_docker/pull/2201/commits/0afd8a751d66b41bd7150d9f9a1bda098a615646#diff-99eaf56c24f9220cbfdae9df3ae1f3a29b7b4b927f88c5168f9791a43b5d74f8R13.

Reply:
I have tested that the optional segment is optional when testing locally, but maybe there should be a better e2e test.